packages/ckeditor5-image/theme/image.css

```diff
@@ -10,7 +10,7 @@
 		text-align: center;
 
 		/* Make sure there is some space between the content and the image. Center image by default. */
-		margin: 1em auto;
```

Comment:
We can't use `var(--ck-spacing-large)` in [content styles](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/content-styles.html) because they could be used in the front-end where CKEditor is not loaded at all and there are no custom property definitions :(

Same with other custom properties such as colors, fonts, etc.

Reply:
You're right, thanks.
I guess we should just stay with the fixed values (committed).
The other way was to override the content rule in the internal editor CSS stylesheet, but I guess the differences is displaying content in the editor and on the webpage may be confusing.

Reply:
Something else came to my mind.
We can also use the fallback value for the variable:
`margin: var(--ck-spacing-large, 0.9em) auto`.
This way it will work properly in the editor and outside of it. But I'm not sure if we gain anything from it. It introduces some external variables in content styles (another thing the integrator has to understand) and doesn't say more than a good comment about why this code looks like this.